spec.html

```diff
@@ -12159,7 +12160,7 @@ <h1>Agents</h1>
     </emu-note>
 
     <emu-clause id="sec-agentsignifier" type="abstract operation">
-      <h1>AgentSignifier ( ): an opaque value used to identify an Agent</h1>
+      <h1>AgentSignifier ( ): an agent signifier</h1>
```

Comment:
If we want to use this term, we should `<dfn>` it and change the description of the `[[Signifier]]` field. Not sure that's in scope for this PR though, and the type we have currently works well enough for now.

Reply:
We've been using "an agent signifier" as a parameter-type since the memory model was added. Is there a higher bar for using it as a return-type? In the absence of a `<dfn>` (for now), surely we should at least use one of the existing terms for it rather than invent a new one. E.g.:
- Agent Record's `[[Signifier]]` says it's "a globally-unique value".
- Agent Events Record's `[[AgentSignifier]]` says it's "a value that admits equality testing".
- AddWaiter, RemoveWaiter, SuspendAgent, and NotifyWaiter say it's "an agent signifier".

Reply:
I am fine with using a term like "agent signifier" as long as it's obvious (it's not), self-describing (also not), or there's a `<dfn>` for it. Already having something that's none of those as a parameter type is unfortunate, but I don't think we should further propagate it without adding a `<dfn>`. It should be pretty simple to just use the description here in a `<dfn>`.